From david-b@pacbell.net  Tue Feb 24 21:56:50 2009
From: David Brownell <david-b@pacbell.net>
Date: Wed, 11 Feb 2009 22:31:12 -0800
Subject: USB: gadget: fix build error in omap_apollon_2420_defconfig
To: Kyungmin Park <kmpark@infradead.org>, Greg KH <greg@kroah.com>
Cc: Tony Lindgren <tony@atomide.com>, "Russell King - ARM Linux" <linux@arm.linux.org.uk>
Message-ID: <200902112231.12887.david-b@pacbell.net>
Content-Disposition: inline


In apollon case, it only used udc, so udc configuration should select
USB_OTG_UTILS also.

Signed-off-by: Kyungmin Park <kyungmin.park@samsung.com>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/gadget/Kconfig
+++ b/drivers/usb/gadget/Kconfig
@@ -191,6 +191,7 @@ config USB_GADGET_OMAP
 	boolean "OMAP USB Device Controller"
 	depends on ARCH_OMAP
 	select ISP1301_OMAP if MACH_OMAP_H2 || MACH_OMAP_H3 || MACH_OMAP_H4_OTG
+	select USB_OTG_UTILS if ARCH_OMAP
 	help
 	   Many Texas Instruments OMAP processors have flexible full
 	   speed USB device controllers, with support for up to 30
